src/com/dynatrace/loadrunner/logic/CFilePatcher.java

```diff
@@ -20,16 +19,15 @@
 	private CFileScanner scanner;
 
 	private LinkedList<String> transactionNames;
-	private ScriptFile scriptFile;
 	private boolean insertCalls;
+	private String scriptName;
 
 	public CFilePatcher(File sourceFile, File targetFile, ScriptFile scriptFile)
 	{
 		writer=null;
 		transactionNames=new LinkedList<String>();
 		this.targetFile=targetFile;
 		this.sourceFile=sourceFile;
-		this.scriptFile=scriptFile;
 	}
 
 	/**
```

Comment:
remove this javadoc or describe it properly
DO NOT USE @author annotation anywhere in the whole project to avoid mentioning people that aren't involved in that project for a long time
using @throws annotation expect to provide a description of the case when that exception might be thrown; otherwise - remove, as it doesn't explain anything